examples/tab_autocompletion.py

```diff
@@ -109,6 +109,18 @@ def instance_query_actors(self) -> List[str]:
         """Simulating a function that queries and returns a completion values"""
         return actors
 
+    def instance_query_movie_ids(self) -> List[str]:
```

Comment:
It would be better if sub-commands like "video movies add" and "video movies delete" actually mutated the internal list of movies (as viewable by "video movies list").

As is, the example does a great job of demonstrating how tab completion and autocompletion works.  But the sub-commands don't behave in the way I would expect.

At the very least, put a big honking warning at the top of the file that lets users know the subcommands don't actually do what their name implies and exist solely to test tab (auto)completion.  But it would be even better if they actually did what their name suggests.